src/client/app/utils/determineCompatibleUnits.ts

```diff
@@ -176,7 +177,10 @@ export function getMeterMenuOptionsForGroup(defaultGraphicUnit: number, deepMete
 
 	// We want the options sorted by meter identifier.
 	// Had to make item.label? potentially undefined due to start up race conditions
-	return sortBy(options, item => item.label?.toLowerCase(), 'asc');
+	// TODO: change second argument of locale from undefined to current language if needed, but alphabetical ordering works
```

Comment:
See comment in uiSelectors.ts about passing the locale. If it works there then I suspect it could similarly be done in create/edit for the group modal component.

Reply:
Finished implementation of changes.